tests/utils/test_files.py

```diff
@@ -64,3 +74,46 @@ def test_rename_file_original_does_not_exist(tmp_path: Path):
     # THEN a FileNotFoundError should be raised
     with pytest.raises(FileNotFoundError):
         rename_file(file_path=file_path, renamed_file_path=renamed_file_path)
+
+
+def test_get_creation_date(tmp_path_factory, timestamp_now: datetime):
+    """Test to retrieve the creation date of a directory."""
+
+    # GIVEN a directory that is created now
+    directory_path: Path = tmp_path_factory.mktemp("some_dir")
+
+    # WHEN retrieving the creation time stamp
+    creation_time_stamp: float = get_directory_creation_time_stamp(directory_path)
+    # THEN the creation time stamp is returned
+    assert isinstance(creation_time_stamp, float)
+
+
+def test_get_all_directories_in_path(
+    path_with_directories_and_a_file: Path, sub_dir_names: List[str], some_file: str
+):
+    """Test get all directories in path."""
+    # GIVEN a path that contains directories and a file
+
+    # WHEN retrieving all directories in the path
+    directories: List[Path] = get_directories_in_path(path=path_with_directories_and_a_file)
+
+    # THEN all directories are returned
+    for sub_dir_name in sub_dir_names:
+        assert sub_dir_name in [directory.name for directory in directories]
+
+    # THEN no files are returned
+    assert some_file not in [directory.name for directory in directories]
```

Comment:
Is this `some_file` in the path? Maybe an assertion in the GIVEN section could be added for clarity